_posts/en/newsletters/2024-06-07-newsletter.md

```diff
@@ -165,17 +165,45 @@ Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
 [Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
 repo], and [BINANAs][binana repo]._
 
-FIXME:Gustavojfe
-
-- [Core Lightning #7252][] lightningd: don't ignore fee limits on mutual close.
-
-- [LDK #2931][] log-outbound-channels
-
-- [Rust Bitcoin #2644][] Add HMAC Extract-and-Expand Key Derivation Function
-
-- [BIPs #1541][] BIP431: Opt In Topologically Restricted Until Confirmation Transactions For More Robust Fee-bumping
-
-- [BIPs #1556][] BIP 337: Compressed Transactions
+- [Core Lightning #7252][] changes the behavior of `lightningd` to ignore the
+  `ignore_fee_limits` setting during a cooperative channel closure. This fixes
+  an issue where a Core Lightning (CLN) channel opener node overpays fees when
+  the counterparty is a LDK node. In this scenario, when the LDK non-opener node
+  (Alice) initiates a cooperative channel closure and begins fee negotiation,
+  the CLN opener node (Bob) responds that the fee can be anything between
+  `min_sats` and `max_channel_size` due to the `ignore_fee_limits` setting. Bob
+  picks the upper bound (contrary to the BOLTs specification), and Alice
+  accepts, resulting in Alice broadcasting a transaction with considerably
+  overpaid fees.
+
+- [LDK #2931][] enhances the logging during path finding to include additional
+  data about direct channels such as whether they’re missing, their minimum
+  [HTLC][topic htlc] amount, and their maximum HTLC amount. The added logging
+  aims to better troubleshoot routing issues by providing visibility into the
+  available liquidity and limitations on each channel.
+
+- [Rust Bitcoin #2644][] adds HKDF (HMAC (Hash-based Message Authentication
+  Code) Extract-and-Expand Key Derivation Function) to the `bitcoin_hashes`
+  component to implement BIP324 in Rust Bitcoin. HKDK is used to derive
+  cryptographic keys from a source of keying material in a secure and
+  standardized way. [BIP324][topic v2 p2p transport] (also known as v2 P2P
+  transport) is a proposal to allow Bitcoin nodes to communicate with each other
+  over encrypted connections (enabled by default in Bitcoin Core). 
+
+- [BIPs #1541][] adds BIP431 to propose [Topologically Restricted Until
+  Confirmation (TRUC)][topic v3 transaction relay] (v3 transactions) which are a
+  subset of standard transactions with additional rules designed to allow
+  transaction replacement while minimizing the cost of overcoming
+  [transaction-pinning][topic transaction pinning] attacks. See Newsletters
+  [#289][news289 v3], [#296][news296 v3] and [#301][news301 v3] for more on v3
+  transactions.
+
+- [BIPs #1556][] adds BIP 337 to propose Compressed Transactions, a
+  serialization protocol to compress bitcoin transactions to reduce their size
+  by up to 50%. They are practical for low-bandwidth transmission such as by
+  satellite, HAM radio, or through steganography. Two RPC commands are proposed:
```

Comment:
```suggestion
  satellite, HAM radio, or steganography. Two RPC commands are proposed:
```

Reply:
Leaving as-is.  I think "through" makes sense here since it is not a complete transmission method itself.

Reply:
That's subtle! But makes sense 👍